### DISCUSSION TITLE
Form组件对于嵌套数组无法正确渲染initialValues

### DISCUSSION THREAD

**Discussion by S-TE11A** [ant-design/ant-design#44547] [General]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/vigorous-goldstine-vsmpm2?file=/index.js)

### Steps to reproduce

运行demo

### What is expected?

四个input中应当按照intialValue中展示对应值：a/b/c/d

### What is actually happening?

input展示为空

| Environment | Info |
| --- | --- |
| antd | 5.8.5 |
| React | 18.1.0 |
| System | macOS Monterey 12.4 |
| Browser | Chrome 版本 116.0.5845.110（正式版本） (x86_64) |

---
可能是使用问题，希望得到社区的帮助，想知道如何正确使用Form接管这样的数据结构

**Comment by MadCcc** [comment]
name 不能响应 `items[index]` 这种。
请使用 Form.List，参考：https://ant.design/components/form-cn#formlist

**Reply by S-TE11A** [reply]
请问Form.List中能否拿到每个Item的value呢？ 可能需要用来做一些判断来展示不同的Form.Item

**Comment by zombieJ** [comment]
不要混合 state 和 FormStore，你始终应该通过 FormStore 来控制展示。一般表格控制是将表格作为自定义组件做，另一种是表格作为 Form 完整内容控制：https://codesandbox.io/s/goofy-shamir-ftrf3r?file=/index.js

**Comment by S-TE11A** [comment]
感谢各位 使用Form.List解决了问题
